src/explorer/SiteTreeItem.ts

```diff
@@ -124,11 +124,22 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             }
         }
         cancelWebsiteValidation(this);
+        // tslint:disable-next-line:strict-boolean-expressions
+        if (workspace.workspaceFolders && workspace.workspaceFolders.length === 1) {
```

Comment:
It's tricky to combine the logic here.
1. Either way, I have to check for `workspace.workspaceFolders` or TS will complain about possibily undefined unless I pass the activeWorkspace, but that feels wrong.
1. I need to check if deployed path is either the active workspace or a subpath of the workspace, but before this point, we don't definitive fspath since it is prompted for later.
1. We could move the logic, but it might affect other things 😕